infra/helm/meshdb/values.yaml

```diff
@@ -92,7 +106,13 @@ pelias:
   port: 3000
   podSecurityContext: {}
   securityContext: {}
-  resources: {}
+  resources:
+    limits:
+      cpu: 1
+      memory: 512Mi
+    requests:
+      cpu: 250m
+      memory: 250Mi
```

Comment:
No fucking idea here. Can probably get away with pinning limit to the smaller request here tho.